consensus/src/quorum_store/utils.rs

```diff
@@ -494,7 +494,13 @@ impl ProofQueue {
                             cur_unique_txns
                                 + txn_summaries
                                     .iter()
-                                    .filter(|txn_summary| !filtered_txns.contains(txn_summary))
+                                    .filter(|txn_summary| {
+                                        !filtered_txns.contains(txn_summary)
+                                                // latest_block_timestamp is microseonds since UNIX epoch
+                                                // expiration_timestamp_secs is seconds since UNIX epoch
+                                            && (txn_summary.expiration_timestamp_secs * 1_000_000)
```

Comment:
Can you add a const to this file, similar to https://github.com/aptos-labs/aptos-core/blob/533f3c33aa0c581da11bf3a14276be8fd40e19c8/crates/aptos/src/node/mod.rs#L59 ?

Reply:
DONE.